spec/models/auth_token_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe AuthToken do
+  it 'encodes and decodes tokens' do
```

Comment:
Capture what the encode and the decode should look like, then test that. Right now the test is technically dependent on the one, so if both the encode and the decode don't do anything, we're still positive.

Different language and context (Python and Encryption), but same concept - https://github.com/laslabs/odoo-base/blob/9.0/fields_encrypted/tests/test_fields_encrypted_text.py#L16

Note I have a `text_enc` and `text` var. Both are `Text` just in an encoded and decoded format. Tests are explicit against those forms